diff --git a/L1Trigger/L1TNtuples/python/customiseL1CustomReco.py b/L1Trigger/L1TNtuples/python/customiseL1CustomReco.py
index 5f9e312..dfc6e3b 100644
--- a/L1Trigger/L1TNtuples/python/customiseL1CustomReco.py
+++ b/L1Trigger/L1TNtuples/python/customiseL1CustomReco.py
@@ -14,25 +14,23 @@ def L1NtupleCustomReco(process):
     # load JEC from SQLite file
     process.load("CondCore.DBCommon.CondDBCommon_cfi")
 
-    ## process.jec = cms.ESSource(
-    ##     "PoolDBESSource",
-    ##     DBParameters = cms.PSet(
-    ##         messageLevel = cms.untracked.int32(0)
-    ##         ),
-    ##     timetype = cms.string('runnumber'),
-    ##     toGet = cms.VPSet(
-    ##         cms.PSet(
-    ##             record = cms.string('JetCorrectionsRecord'),
-    ##             tag    = cms.string('JetCorrectorParametersCollection_Summer15_25nsV6_DATA_AK4PFchs'),
-    ##             label  = cms.untracked.string('AK4PFCHS')
-    ##             ),
-    ##         ), 
-    ##     connect = cms.string('sqlite:Summer15_25nsV6_DATA.db')
-    ##     # uncomment above tag lines and this comment to use MC JEC
-    ##     # connect = cms.string('sqlite:Summer12_V7_MC.db')
-    ##     )
-    ## 
-    ## process.es_prefer_jec = cms.ESPrefer('PoolDBESSource','jec')
+    process.jec = cms.ESSource(
+        "PoolDBESSource",
+        DBParameters = cms.PSet(
+            messageLevel = cms.untracked.int32(0)
+            ),
+        timetype = cms.string('runnumber'),
+        toGet = cms.VPSet(
+            cms.PSet(
+                record = cms.string('JetCorrectionsRecord'),
+                tag    = cms.string('JetCorrectorParametersCollection_Fall15_25nsV2_MC_AK4PFchs'),
+                label  = cms.untracked.string('AK4PFchs')
+                ),
+            ),
+        connect = cms.string('sqlite:Fall15_25nsV2_MC.db')
+        )
+
+    process.es_prefer_jec = cms.ESPrefer('PoolDBESSource','jec')
     
     # re-apply JEC for AK4 CHS PF jets
     process.load('JetMETCorrections.Configuration.JetCorrectors_cff')
